static/index-web.html

```diff
@@ -62,6 +62,10 @@
       }
     </style>
 
+    <link rel="manifest" href="./public/pwa/manifest.json"/>
```

Comment:
I'd like to inject these in index.js conditional on SITE_NAME
We can make a specific DESCRIPTION env if necessary.